doc/Command-Reference.md

```diff
@@ -3981,10 +3991,15 @@ kindly use, double tab i.e. <tab><tab> to see the available breakout option cust
   admin@sonic:~$ sudo config interface breakout  Ethernet0 <tab><tab>
   <tab provides option for breakout mode>
   1x100G[40G]  2x50G        4x25G[10G]
+  ```
 
+  This command also provides  "--force-remove-dependencies/-f" option to CLI, which will remove the configuration dependencies automatically.
```

Comment:
Suggest elaborating here that Yang models are used to automatically determine and remove configuration dependencies.

Reply:
Modified as suggested. 👍